Source/SelfService/Web/microservice/base/create.tsx

```diff
@@ -200,7 +200,7 @@ export const Create: React.FunctionComponent<Props> = (props) => {
                 </Grid>
 
                 <Grid item>
-                    <Typography component="p" sx={styles.data}>
+                    <Typography sx={styles.data}>
```

Comment:
i cant find any code that sets default typography component to p. Is this resolved by the library itself on runtime? curious since it looks cleaner this way